test/compress/if_return.js

```diff
@@ -205,3 +248,56 @@ issue_1089: {
         }
     }
 }
+
+issue_1437: {
+    options = {
+        if_return    : true,
+        sequences    : true
```

Comment:
Can you add the option `conditionals : false,` to this test just to emphasize where the bug existed in the 2.7.5 release?